src/d/d_font_manager.cpp

```diff
@@ -0,0 +1,77 @@
+#include <d/d_font_manager.h>
+#include <d/d_heap.h>
+#include <sized_string.h>
+
+static dFontMng_c l_dFontMng_obj[3];
+
+static char *systemFonts[] = {"normal_00.brfnt", "special_00.brfnt", "picture_00.brfnt"};
+static char *fonts[] = {"normal_00.brfnt", "normal_01.brfnt", "special_00.brfnt", "special_01.brfnt",
+        "picture_00.brfnt"};
```

Comment:
my intellisense complains about this not being const char. When added it doesn't seem to change anything, not sure if I am missing something or not